e2e/testcases/acme_test.go

```diff
@@ -190,6 +192,18 @@ func TestAcmeCorpRepo(t *testing.T) {
 	nt.WaitForRepoSyncs()
 }
 
+// TestObjectInCMSNamespace will test that user can sync object to CMS namespace
+func TestObjectInCMSNamespace(t *testing.T) {
+	nt := nomostest.New(t, ntopts.SkipMonoRepo, ntopts.Unstructured)
+
+	nt.RootRepos[configsync.RootSyncName].Copy("../../examples/object-in-cms-namespace", "acme")
+	nt.RootRepos[configsync.RootSyncName].CommitAndPush("adding resource to config-management-system namespace")
+	nt.WaitForRepoSyncs()
+	namespace := configmanagement.ControllerNamespace
+	checkResourceCount(nt, kinds.Role(), namespace, 1, nil, nil)
+	checkResourceCount(nt, kinds.ConfigMap(), namespace, 1, map[string]string{"app.kubernetes.io/managed-by": "configmanagement.gke.io"}, nil)
```

Comment:
Instead of checking the resource count, why not checking the particular object? That will ensure the resource is the one you expected.

Reply:
validation has been changed to use `nt.validate` directly